packages/api/src/resolvers/subscriptions/index.ts

```diff
@@ -434,37 +427,60 @@ export const scanFeedsResolver = authorized<
   }
 
   if (!url) {
+    log.error('Missing opml and url')
+
     return {
       errorCodes: [ScanFeedsErrorCode.BadRequest],
     }
   }
 
   try {
-    // fetch HTML and parse feeds
-    const response = await axios.get(url, {
-      timeout: 5000,
-      headers: {
-        'User-Agent': 'Mozilla/5.0',
-        Accept: 'text/html',
-      },
-    })
-    const html = response.data as string
-    const dom = parseHTML(html).document
-    const links = dom.querySelectorAll('link[type="application/rss+xml"]')
-    const feeds = Array.from(links)
-      .map((link) => ({
-        url: link.getAttribute('href') || '',
-        title: link.getAttribute('title') || '',
-        type: 'rss',
-      }))
-      .filter((feed) => feed.url)
+    // fetch page content and parse feeds
+    const response = await axios.get(url, RSS_PARSER_CONFIG)
+    const content = response.data as string
+    // check if the content is html or xml
+    const contentType = response.headers['content-type']
+    const isHtml = contentType?.includes('text/html')
```

Comment:
I wonder if we will need more types here. Will find out with some testing I guess.